lp-builder-config/openstack.yaml

```diff
@@ -295,6 +295,16 @@ projects:
     launchpad: charm-cinder-lvm
     repository: https://opendev.org/openstack/charm-cinder-lvm.git
 
+  - name: OpenStack Cinder Nimblestorage Charm
+    charmhub: cinder-nimblestorage
+    launchpad: charm-cinder-nimblestorage
+    repository: https://opendev.org/openstack/charm-cinder-nimblestorage.git
+  
+  - name: OpenStack Cinder Solidfire Charm
+    charmhub: cinder-solidfire
+    launchpad: charm-cinder-solidfire
+    repository: https://opendev.org/openstack/charm-cinder-solidfire.git
```

Comment:
Before merging, I just need to ask/verify that these two charms do support all the versions specified in the default branches above.  If not, then they need scoping directly here, please.  Thanks.

Reply:
Thank you!